home-page.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Beyond the Reel</title>
+
+    <!--Will add an icon-->
+    <link rel="shortcut icon" href="" type="" />
```

Comment:
If this isn't done, comment it out or delete it and make a todo ticket for later